app/api/BaseApi.js

```diff
@@ -54,18 +59,39 @@ class BaseApi {
     }
   }
 
-  static getHeader = async () => {
-    const authToken = await AsyncStorage.getItem('AUTH_TOKEN');
-
+  static getHeader = (token) => {
```

Comment:
@limkimsan please help rename this method name to the appropriate name, Header is a bit broader while our implementation code is about authorization so I would suggest to rename from getHeader(token) to generateAuthorizationHeader(token)